web/lib/plugins/sitemap.js

```diff
@@ -0,0 +1,93 @@
+const sitemap = require('sitemap');
+
+exports.register = (server, options, next) => {
+  const port = process.env.PORT || 8000;
+  const host = process.env.HOSTNAME || '127.0.0.1';
+  const protocol = process.env.PROTOCOL || 'http';
+  const hostWithPort = `${protocol}://${host}:${port}`;
+  let currentSitemap;
+  const skeleton = {
+    hostname: hostWithPort,
+    cacheTime: 0, // We manage the reset of the cache manually
+    urls: [],
+  };
+  const staticUrls = [
```

Comment:
This could be generated at first load of the application through a configuration (but should be kept between each refresh)
To add as a comment ?

Reply:
How about